code/style.css

```diff
@@ -1,4 +1,243 @@
+  justify-content: space-evenly;
+  background: lightgreen;
+ }
+
+
+ .title {
+display: flex;
+align-items: inherit;
+justify-content: space-evenly;
+margin: 5px;
+}
+
+.title h1 {
+  order: 2;
+  margin-right: 10px;
+  margin-left: 35px;
+}
+
+.title img {
+  width: 50px;
+  margin: 10px;
+  order: 1;
+} 
+
+.nav a {
+  color: rgb(17, 15, 15);
+  text-decoration: solid;
+  margin-left: 16px;
+  justify-content: space-between;
+  text-decoration-line: none;
+  color: black;
+  }
+
+  .nav a:hover {
+    color: orange;
+  }
+
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+  }
+
+.hamburger div {
+  width: 30px;  
+  height: 5px;
+  background-color: black;
+  border-radius: 5px;
+  }
+
+/* COLUMN-LAYOUT HEAD NEWS */
+
+.column-layout {
+  max-width: 1300px;
+  margin: 40px auto 0px auto;
+  line-height: 1.65;
+  background: coral;
+  padding: 20px;
+  border: 2px solid orangered;
+  border-radius: 5px;
+  padding: 5px;
+  box-shadow: 6px 6px 0 0 orange;
+}
+
+.main-column a {
+  text-decoration-line: none;
+  color: black;
+}
+
+.main-column a:hover {
+color: orange;
+}
+
+.main-column {
+  flex: 1;
+  order: 1;
+  padding: 10px;
+}
+
+.bird-column {
+margin: 5px auto;
+border-radius: 5px;
+padding: 5px;
+box-shadow: 6px 6px 0 0 orange;
+flex: 1;
+order: 2;
+}
+
+.bird-column:hover {
+  animation: shake 0.6s;
+
+  animation-iteration-count: infinite;
+}
+
+@keyframes shake {
+  0% { transform: translate(1px, 1px) rotate(0deg); }
+  10% { transform: translate(-1px, -2px) rotate(-1deg); }
+  20% { transform: translate(-3px, 0px) rotate(1deg); }
+  30% { transform: translate(3px, 2px) rotate(0deg); }
+  40% { transform: translate(1px, -1px) rotate(1deg); }
+  50% { transform: translate(-1px, 2px) rotate(-1deg); }
+  60% { transform: translate(-3px, 1px) rotate(0deg); }
+  70% { transform: translate(3px, 1px) rotate(-1deg); }
+  80% { transform: translate(-1px, -1px) rotate(1deg); }
+  90% { transform: translate(1px, 2px) rotate(0deg); }
+  100% { transform: translate(1px, -2px) rotate(-1deg); }
+}
+
+/* CARDS INFO ARTICELS */
+
+.colum-cards {
+  max-width: 1400px;
+  margin: 40px auto 0 auto;
+  }
+  
+  .cards {
+  padding:20px;
+  margin: 20px auto;
+  flex-basis: 90%;
+  box-sizing: border-box;
+  border-radius: 3px;
+  box-shadow: 6px 6px 0 0 orange;
+  }
+
+  .cards a {
+      text-decoration-line: none;
+      color: black;
+  }
+
+
+.cards a:hover {
+  color: orange;
+  }
+
+.cards:nth-child(1){background-color: rgb(192, 153, 55);}
+.cards:nth-child(2){background-color: lightblue;}
+.cards:nth-child(3){background-color: rgb(175, 221, 161);}
+.cards:nth-child(4){background-color: rgb(188, 170, 238);}
+.cards:nth-child(5){background-color: rgb(192, 153, 55);}
+.cards:nth-child(6){background-color: lightblue;}
+.cards:nth-child(7){background-color: rgb(175, 221, 161);}
+.cards:nth-child(8){background-color: rgb(188, 170, 238);}
```

Comment:
Take a look at this resource and see if you can take away some lines here: https://css-tricks.com/useful-nth-child-recipies/#select-every-fourth-starting-at-the-first :)